test/integration/src/py/test_sifnode_peggy1_pause_unpause.py

```diff
@@ -0,0 +1,114 @@
+from typing import Tuple
```

Comment:
Why is this test labeled as a peggy1 test? Why would it not work on Peggy2

Reply:
Agreed, updated